en/micro-integrator/docs/administer-and-observe/logs/configuring_log4j_properties.md

```diff
@@ -1,94 +1,625 @@
+    appender.CARBON_LOGFILE.fileName = ${sys:carbon.home}/repository/logs/wso2carbon.log
+    appender.CARBON_LOGFILE.filePattern = ${sys:carbon.home}/repository/logs/wso2carbon-%d{MM-dd-yyyy}.log
+    appender.CARBON_LOGFILE.layout.type = PatternLayout
+    appender.CARBON_LOGFILE.layout.pattern = [%d] %5p {%c} - %m%ex%n
+    appender.CARBON_LOGFILE.policies.type = Policies
+    appender.CARBON_LOGFILE.policies.time.type = TimeBasedTriggeringPolicy
+    appender.CARBON_LOGFILE.policies.time.interval = 1
+    appender.CARBON_LOGFILE.policies.time.modulate = true
+    appender.CARBON_LOGFILE.policies.size.type = SizeBasedTriggeringPolicy
+    appender.CARBON_LOGFILE.policies.size.size=10MB
+    appender.CARBON_LOGFILE.strategy.type = DefaultRolloverStrategy
+    appender.CARBON_LOGFILE.strategy.max = 20
+    appender.CARBON_LOGFILE.filter.threshold.type = ThresholdFilter
+    appender.CARBON_LOGFILE.filter.threshold.level = DEBUG
+    ```
 
-    ```xml
-    logger.VFSTransportSender.name = org.apache.synapse.transport.vfs.VFSTransportSender
-    logger.VFSTransportSender.level = WARN
+-   `ERROR_LOGFILE`: This is a RollingFile appender that print the error logs to the `<MI_HOME>/repository/logs/wso2error.log` file.
+
+
+    ```xml tab='ERROR_LOGFILE'
+    # Appender config to SERVICE_APPENDER
+    appender.ERROR_LOGFILE.type = RollingFile
+    appender.ERROR_LOGFILE.name = ERROR_LOGFILE
+    appender.ERROR_LOGFILE.fileName = ${sys:carbon.home}/repository/logs/wso2error.log
+    appender.ERROR_LOGFILE.filePattern = ${sys:carbon.home}/repository/logs/wso2error-%d{MM-dd-yyyy}.log
+    appender.ERROR_LOGFILE.layout.type = PatternLayout
+    appender.ERROR_LOGFILE.layout.pattern = [%d] %5p {%c} - %m%ex%n
+    appender.ERROR_LOGFILE.policies.type = Policies
+    appender.ERROR_LOGFILE.policies.time.type = TimeBasedTriggeringPolicy
+    appender.ERROR_LOGFILE.policies.time.interval = 1
+    appender.ERROR_LOGFILE.policies.time.modulate = true
+    appender.ERROR_LOGFILE.policies.size.type = SizeBasedTriggeringPolicy
+    appender.ERROR_LOGFILE.policies.size.size=10MB
+    appender.ERROR_LOGFILE.strategy.type = DefaultRolloverStrategy
+    appender.ERROR_LOGFILE.strategy.max = 20
+    appender.ERROR_LOGFILE.filter.threshold.type = ThresholdFilter
+    appender.ERROR_LOGFILE.filter.threshold.level = WARN
     ```
 
-2.  Then, add the logger to the list of loggers by referring the logger name:
+### Service Logs
 
-    ```xml
-    loggers = AUDIT_LOG, SERVICE_LOGGER, VFSTransportSender, 
-    ```
+This logger generates logs for services deployed in the Micro Integrator. It refers the `SERVICE_LOGFILE` appender and prints logs to the `MI_HOME/repository/logs/wso2-mi-service.log` file.
+
+!!! Note
+    If you want to have separate log files for individual services, you need to add loggers for each service and then specify appenders for the loggers.
+    
+```xml tab='SERVICE_LOGGER'
+logger.SERVICE_LOGGER.name= SERVICE_LOGGER
```

Comment:
To enable per service log for TestProxy (a proxy service) the configuration should be as following, `logger.SERVICE_LOGGER.name = SERVICE_LOGGER.TestProxy`. Shall we include this as well?

Reply:
added a note about this...